python/lsst/pipe/tasks/postprocess.py

```diff
@@ -789,6 +791,181 @@ def _makeArgumentParser(cls):
+        """No config to persist, so override to remove config persistance.
+        """
+        pass
+
+    def runDataRef(self, dataRefList):
+        visit = dataRefList[0].dataId['visit']
+
+        self.log.debug("Concatenating metadata from %d per-detector calexps (visit %d)" %
+                       (len(dataRefList), visit))
+
+        expCatalog = self._combineExposureMetadata(visit, dataRefList, isGen3=False)
+
+        dataRefList[0].put(expCatalog, 'visitSummary', visit=visit)
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        dataRefs = butlerQC.get(inputRefs.calexp)
+        visit = dataRefs[0].dataId.byName()['visit']
+
+        self.log.debug("Concatenating metadata from %d per-detector calexps (visit %d)" %
+                       (len(dataRefs), visit))
+
+        expCatalog = self._combineExposureMetadata(visit, dataRefs)
+
+        butlerQC.put(expCatalog, outputRefs.visitSummary)
+
+    def _combineExposureMetadata(self, visit, dataRefs, isGen3=True):
+        """Make a combined exposure catalog from a list of dataRefs.
+
+        Parameters
+        ----------
+        visit : `int`
+            Visit identification number
+        dataRefs : `list`
+            List of calexp dataRefs in visit.  May be list of
+            `lsst.daf.persistence.ButlerDataRef` (Gen2) or
+            `lsst.daf.butler.DeferredDatasetHandle` (Gen3).
+        isGen3 : `bool`, optional
+            Specifies if this is a Gen3 list of datarefs.
+
+        Returns
+        -------
+        visitSummary : `lsst.afw.table.ExposureCatalog`
+            Exposure catalog with per-detector summary information.
+        """
+        schema = afwTable.ExposureTable.makeMinimalSchema()
+        schema.addField('visit', type='I', doc='Visit number')
+        schema.addField('detector_id', type='I', doc='Detector number')
+        schema.addField('physical_filter', type='String', size=32, doc='Physical filter')
+        schema.addField('band', type='String', size=32, doc='Name of band')
+        schema.addField('psfSigma', type='F',
+                        doc='PSF model second-moments determinant radius (center of chip) (pixel)')
+        schema.addField('psfArea', type='F',
+                        doc='PSF model effective area (center of chip) (pixel**2)')
+        schema.addField('psfIxx', type='F',
+                        doc='PSF model Ixx (center of chip) (pixel**2)')
+        schema.addField('psfIyy', type='F',
+                        doc='PSF model Iyy (center of chip) (pixel**2)')
+        schema.addField('psfIxy', type='F',
+                        doc='PSF model Ixy (center of chip) (pixel**2)')
+        schema.addField('raCorners', type='ArrayD', size=4,
+                        doc='Right Ascension of bounding box corners (degrees)')
+        schema.addField('decCorners', type='ArrayD', size=4,
+                        doc='Declination of bounding box corners (degrees)')
```

Comment:
I do wish we had a better way to store these corners, but I can't think of one off hand.